src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -284,30 +286,12 @@ export const InRequestEditPanel: FunctionComponent<IInRequestEditPanel> = (
               <Controller
                 name="empType"
                 control={control}
-                rules={{
-                  required: "Employee Type is required",
-                }}
-                render={({ field: { onBlur, onChange, value } }) => (
+                render={({ field }) => (
                   <RadioGroup
+                    {...field}
                     id="empTypeId"
-                    onBlur={onBlur}
-                    value={value}
-                    onChange={(e, option) => {
-                      /* If they change employee type, clear out the related fields */
-                      setValue("gradeRank", "");
-                      if (option.value === EMPTYPES.Contractor) {
-                        setValue("isNewCivMil", "");
-                        setValue("prevOrg", "");
-                        setValue("isNewToBaseAndCenter", "");
-                        setValue("isTraveler", "");
-                        setValue("isSupervisor", "");
-                      } else {
-                        setValue("hasExistingCAC", "");
-                        setValue("CACExpiration", undefined);
-                      }
-                      onChange(e, option);
-                    }}
                     aria-describedby="empTypeErr"
+                    disabled={true}
                     layout="horizontal"
                   >
```

Comment:
I debated initially how much to pull out and how much to leave behind.  Ultimately, I decided to pull out the logic of what happens on a field changing, as if we go to remove the disabled, we should go to the new form to see what new logic we may need to add in (and/or we are going to be updating based on "what" the changes is, rather then assuming it is good with what is there.  I thought about replacing with just text, but thought we want them to know that it was a field they initially selected/input and what the "other" options were if a radio selection.  Also, I had realized that we call the update which takes the form data and then passes it to a transform.  It assumes it is a IInRequest so certain fields are required.  Rather than redoing all that logic now, felt it was worth just leaving it as is, so that it just passes along the current value to the update, rather then injecting the values manually back into the object, or changing what is required for an update.  Ideally we add all these back in as "non" disabled and have coded logic to handle each scenario.